commands.go

```diff
@@ -41,7 +41,7 @@ func (cmd *ThreadCommand) Command() *imap.Command {
 	return &imap.Command{
 		Name: "THREAD",
 		Arguments: []interface{}{
-			cmd.Algorithm,
+			formatThreadAlgorithm(cmd.Algorithm),
```

Comment:
According to the RFC, this should be an atom (`RawString`), not a quoted string.